include/iarc7_fc_comms/CommonFcComms.hpp

```diff
@@ -97,6 +101,12 @@ namespace FcComms{
             have_new_direction_command_message_ = true;
         }
 
+        inline void contactSwitchMessageHandler(
+                const iarc7_msgs::LandingGearContactsStamped::ConstPtr& message) {
+            last_contact_switch_message_ptr_ = message;
```

Comment:
I don't either now that you mention it. We have been doing it other places as well